src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The `Circle` class should implement methods to calculate the area and set a color, as these are likely required by the task. Ensure that the class has fields for radius and color, and methods to manipulate these fields.